data/assets.json

```diff
@@ -6874,6 +6874,108 @@
     "type": "native"
   },
   {
+    "coingecko": "tether",
+    "description": "Tether from Solana via Picasso",
+    "id": {
+      "osmosis": {
+        "osmosis-1": "ibc/D105950618E47CA2AEC314282BC401625025F80A4F812808DEEBB1941C685575"
+      }
+    },
+    "logo": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/usdt.solana.pica.svg",
+    "name": "Solana USDT on Picasso",
+    "precision": 6,
+    "slugs": [],
+    "symbol": "solana.USDT.pica",
```

Comment:
what is the difference between this and `USDT.sol.pica`?

https://app.osmosis.zone/assets/USDT.sol.pica?sellOpen=false&buyOpen=false

Reply:
we changed the naming scheme around this time to go from
chain.Token.bridge to Token.chain.bridge
was just an older format, have fixed

Reply:
does the ibc denom need an update too?